### PR TITLE
animation:add demo to use cubic-bezier

### PR DESCRIPTION
**Pull request by guoweilkd**
### Description of the feature or fix

the example show the use of cubic-bezier3 in animation.
the control point P1,P2 of cubic-bezier3 can be adjusted by slider.and the chart shows the cubic in real time. you can click run button see animation in current point of cubic-bezier3.

The demo screenshot:
Screenshot

### Checkpoints
- [x] Follow the [styling guide](https://github.com/lvgl/lvgl/blob/master/docs/CODING_STYLE.md)
- [ ] Update [CHANGELOG.md](https://github.com/lvgl/lvgl/blob/master/docs/CHANGELOG.md)
- [ ] Update the documentation

### CODE REVIEW

File: examples/anim/lv_example_anim_3.c [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+/**
+ * create a animation
+ */
+void lv_example_anim_3(void)
+{
+    page_obj_init(lv_scr_act());
+
+    lv_anim_init(&ginfo.a);
+    lv_anim_set_var(&ginfo.a, ginfo.anim_obj);
+    lv_anim_set_values(&ginfo.a, lv_obj_get_x(ginfo.anim_obj), lv_obj_get_x(ginfo.anim_obj) + 400);
```
Comment (kisvegabor) [inline examples/anim/lv_example_anim_3.c]: I'd rather animate between `10` and `lv_obj_get_width(parent) - lv_obj_get_width(ginfo.anim_obj) - 10` to make work well on every screen sizes.

File: examples/anim/lv_example_anim_3.c [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+    lv_slider_set_range(ginfo.p1_slider, 0, 1024);
+    lv_slider_set_range(ginfo.p2_slider, 0, 1024);
+    lv_obj_add_event_cb(ginfo.p1_slider, slider_event_cb, LV_EVENT_VALUE_CHANGED, NULL);
+    lv_obj_add_event_cb(ginfo.p2_slider, slider_event_cb, LV_EVENT_VALUE_CHANGED, NULL);
+
+    ginfo.run_btn = lv_btn_create(par);
+    lv_obj_set_size(ginfo.run_btn, 80, 40);
+    lv_obj_add_event_cb(ginfo.run_btn, run_btn_event_handler, LV_EVENT_CLICKED, NULL);
+    lv_obj_t * btn_label = lv_label_create(ginfo.run_btn);
+    lv_label_set_text(btn_label, "run");
```
Comment (kisvegabor) [inline examples/anim/lv_example_anim_3.c]: `LV_SYMBOL_PLAY`?

**Review by kisvegabor — COMMENTED** [review]
Nice example!

I'm concerned only about its responsiveness. Note that the examples in the docs are running on a 320x240 screen so it should fit into that size.

Probably a `grid` layout would be useful to position the widgets in a flexible way. I can help with that if needed.

**Comment by guoweilkd** [comment]
find a problem, the anim_obj animation  can't run if add it to grid layout. Is this expected?

**Comment by kisvegabor** [comment]
> find a problem, the anim_obj animation can't run if add it to grid layout. Is this expected?

Yes, the layout has a higher priority than the manual x/y coordinate setting. You can animate with `lv_obj_set_style_translate_x(obj, value, LV_PART_MAIN)`

**Comment by kisvegabor** [comment]
Thanks for the fixes. I've just pushed some minor changes.  Do you agree with them?

Are you interested in adding the Micropython version of the example too?

**Comment by guoweilkd** [comment]
> Thanks for the fixes. I've just pushed some minor changes. Do you agree with them?

looks good. I agree.

> Are you interested in adding the Micropython version of the example too?

sorry, I'm not familiar with Micropython.I'll add it when I'm familiar with it.

**Comment by kisvegabor** [comment]
>> Are you interested in adding the Micropython version of the example too?

> sorry, I'm not familiar with Micropython.I'll add it when I'm familiar with it.

No problem. @uraich Would you like to add this example in Micropython?

**Comment by uraich** [comment]
@kisvegabor : I could not find the example in lvgl master yet. I guess it has not been merged yet.
The MP version is at [https://github.com/uraich/lv_mpy_examples_v8/blob/main/anim/lv_example_anim_3.py](https://github.com/uraich/lv_mpy_examples_v8/blob/main/anim/lv_example_anim_3.py)
I will be away for some time. Can you pick the code from my github repository and integrate it into lvgl master as soon as the C version of the program gets merged?

**Comment by kisvegabor** [comment]
@uraich 
Okay, I merge it now and we can add the MP example later. 

> The MP version is at https://github.com/uraich/lv_mpy_examples_v8/blob/main/anim/lv_example_anim_3.py

So that example is ready to merge?

**Comment by kisvegabor** [comment]
Thank you @guoweilkd!

**Comment by uraich** [comment]
@kisvegabor : Since you merged so quickly, I could create a new PR with just the MP version of the example.
@guoweilkd: Nice Demo!

**Pull request merged**